### ISSUE TITLE
[Question] How to install debugging tools into the WSL VM?

### ISSUE THREAD

**Issue by rumstead** [rancher-sandbox/rancher-desktop-wsl-distro#58]
For instance, I want to install `openssl`. Since the package managers are removed, we would either need to mount them or in the case of `openssl` compile the source code in the VM.

**Comment by mook-as** [comment]
Since this is alpine based, just running `apk add -U openssl` should do.  We actually added the package managers back in in 392342277bb9ee2f2527446ceaa1f66254591c9b.

**Comment by rumstead** [comment]
> Since this is alpine based, just running `apk add -U openssl` should do. We actually added the package managers back in in [3923422](https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/commit/392342277bb9ee2f2527446ceaa1f66254591c9b).

Thanks for the quick feedback. Any idea when that will be cut for release and used in Rancher Desktop? Or how can I tell which version of `rancher-desktop-wsl-distro` rancher desktop is using?

**Comment by mook-as** [comment]
That should be already in use; we're currently using [0.31](https://github.com/rancher-sandbox/rancher-desktop/blob/v1.7.0/pkg/rancher-desktop/assets/dependencies.yaml#L5), and the [relevant commit](https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/commit/392342277bb9ee2f2527446ceaa1f66254591c9b) was in 0.21 (according to the list of tags that includes the commit, per GitHub).  It looks like it should have been available as early as [Rancher Desktop 1.3.0](https://github.com/rancher-sandbox/rancher-desktop/blob/v1.3.0/scripts/download/wsl.mjs#L12).

For checking the version you have: Go into the distribution (`wsl -d rancher-desktop`) and just run `cat /etc/os-release` like a normal distribution. I have:
```
NAME="Rancher Desktop WSL Distro"
ID="rancher-desktop-wsl-distro"
PRETTY_NAME="Rancher Desktop WSL Distribution"
VERSION_ID="0.31"
BUILD_ID="9c85a82ec8974592a574fe7de8415cffabb1c2cd"
HOME_URL="https://rancherdesktop.io/"
SUPPORT_URL="https://rancher-users.slack.com/channels/rancher-desktop"
BUG_REPORT_URL="https://github.com/rancher-sandbox/rancher-desktop-wsl-distro/issues/new"
```
The version number is in the `VERSION_ID` field. (And the specific commit is listed as `BUILD_ID`; here I have something built from 9c85a82ec8974592a574fe7de8415cffabb1c2cd.)  Rancher Desktop should be updating it as needed, since a mismatched version would break actually running things.  That's possible because we use an auxiliary `rancher-desktop-data` distribution for all user data.

**Comment by rumstead** [comment]
Thanks! I am working on a Mac troubleshooting someone’s windows device so appreciate the patience with my questions :)
